pkg/subscriptions/main.js

```diff
@@ -617,5 +611,5 @@ require([
         subscriptions.manager = subscription_manager(retrieve_current);
     }
 
-    init();
-});
+    $(init);
```

Comment:
You need to move register_dialog_initialize() into the init function, or call it once the page has loaded.